server/server.js

```diff
@@ -385,17 +385,8 @@ var Server = IgeClass.extend({
 		ige.network.debug(self.isDebugging);
 		// Start the network server
 		ige.network.start(self.port, function (data) {
-			console.log('connecting to BE:', global.beUrl);
-
-			var domain = null;
 
-			// dev gets map from local file
-			if (ige.env == 'standalone' || ige.env == 'standalone-remote' || ige.env === 'production') { // production or staging gets map data from API
-				// using BE's URL instead of GS Manager because GS Manager is overloaded right now so..
-				domain = 'https://www.modd.io';
-			} else {
-				domain = global.beUrl;
-			}
+			var domain = global.beUrl;
```

Comment:
I am not sure what problem is being solved with this.

Reply:
Currently, GS tries to download game json from modd.io for the production environment.
Updating this to download json from gs manager, going forward GS would never send direct request to BE (modd.io) so we can enable bot fight mode to prevent dos attacks. Cloudflare detects GS as a bot and blocks requests.
FYI, I removed self.config.BE_URL || gs_manager.modd.io, since the default config is production config and is being applied for standalone servers as well.

Reply:
So what we need is `production` env domain to be gs_manager.modd.io. This proposed change removes the domain of modd.io for standalone, which will disable fetching games with the flag `--game=:gameId` in standalone, right?

Reply:
No, standalone servers would still work with --game=:gameId, but instead of using modd.io, they will be using gs_manager to download game json as well.
For standalone servers, default config is being applied and is already fetching the production config file.
<img width="489" alt="image" src="https://user-images.githubusercontent.com/17493609/167181720-31429ba8-7f76-4aee-a04b-c00c21621915.png">